modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -111,9 +111,9 @@ zypper in spacewalk-utils
 Adjust the version of the channel names to match your {ubuntu} version:
 +
 ----
-spacewalk-common-channels ubuntu-1804-amd64-main \
-ubuntu-1804-amd64-security \
-ubuntu-1804-amd64-updates
+spacewalk-common-channels -u admin -p pass -a amd64-deb ubuntu-1804-amd64-main \
```

Comment:
Do not add `-u` and `-p`. The script will request them, and they can be different.

Are you sure `-a` is needed, and if so: does it work? When I tested this weeks ago, it worked without it.